src/utils/errors.ts

```diff
@@ -27,8 +27,8 @@ function getSourceCode(node: ASTNode | undefined): string {
   const sourceUnit = node.getClosestParentByType(SourceUnit);
   if (sourceUnit === undefined) return '';
   const filePath = sourceUnit.absolutePath;
-  if (fs.existsSync(filePath)) {
-    const content = fs.readFileSync(filePath, { encoding: 'utf-8' });
+  if (existsSync(filePath)) {
```

Comment:
Initially, I also made it asynchronous, but that made it necessary to make a number of other functions and some class methods also async. Since this, IIRC, is only used at error throwing, in my opinion the benefit is not significant considering all the additional trouble, that's why I kept it like this. Let me know what you think!